### DISCUSSION TITLE
Y axis alignment

### DISCUSSION THREAD

**Discussion by Laul** [patrykandpatrick/vico#567] [Questions]
Hi ! 
I continue the integration of your amazing lib, and would need your help once more 😇 

I am updating my code to update Vico lib to the release 2.0.0.alpha-6. At the same time, I'm polishing some UI such as aligning things. Below is a picture to illustrate my question. 
<img width="183" alt="image" src="https://github.com/patrykandpatrick/vico/assets/2975027/4b286c78-e652-4525-b29c-4bf6b27be41a">

In the picture, the black boxes  are the cartesianChartHost container. I would like that the chart itself be aligned to the top of the cartesianChartHost boxes. Of what I see, it seems that there is a white space between the cartesianChart itself and the top of the cartesianChartHost box. Is there a simple way to remove it? 

I tried to add an alignment info to the Host modifier, but it seems I am unable to find a "compatible" alignment modifier : 
![image](https://github.com/patrykandpatrick/vico/assets/2975027/f07d50a0-f310-4538-83e0-f0cb04933359)
 
I also saw in your code that you define some bounds for the host. Is there a way to override it easily ? 

Thanks a lot for your help !

**Comment by patrickmichalik** [comment]
Hello! This empty space is added so that there’s room for the `Marker`. To remove it, you can update your `Marker` to request no insets. Since composables (unlike views) can draw outside of their bounds, and your chart isn’t at the very top of the screen, the `Marker` will remain visible. Another option, assuming you’re using `MarkerComponent`, is to update to [Vico 2.0.0 Alpha 7](https://github.com/patrykandpatrick/vico/releases/tag/v2.0.0-alpha.7) and apply `MarkerComponent.LabelPosition.AroundPoint`. If you need any help applying either of these solutions, feel free to let me know.
